src/App.tsx

```diff
@@ -1,61 +1,112 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function findUserById(id: number) {
+  return usersFromServer.find(user => user.id === id) || null;
+}
+
+const todoList = todosFromServer.map(todo => (
+  {
+    ...todo,
+    user: findUserById(todo.userId),
+  }
+));
 
 export const App = () => {
+  const [titleForNewTodos, setTitleForNewTodos] = useState('');
```

Comment:
Consider using more descriptive names for state variables, like `newTodoTitle`, `newTodoUserId`, `todos`, `isTitleInvalid`, `isUserInvalid`. It will make the code more readable and self-explanatory. 🤓